src/os/bluestore/KernelDevice.cc

```diff
@@ -187,6 +189,72 @@ void KernelDevice::close()
   path.clear();
 }
 
+static string get_dev_property(const char *dev, const char *property)
+{
+  char val[1024] = {0};
+  get_block_device_string_property(dev, property, val, sizeof(val));
+  return val;
```

Comment:
i'll make the conversion to std::string explicit so that it's clear it happens first.  (I think it does on its... but not really sure actually!)